pallets/creditcoin/src/lib.rs

```diff
@@ -293,30 +287,22 @@ pub mod pallet {
 		/// A deal order has been funded by a lender. This indicates that the lender
 		/// has initiated the actual loan by transferring the loan amount to the borrower
 		/// on an external chain.
-		/// [funded_deal_order_id, funded_deal_order]
-		DealOrderFunded(
-			DealOrderId<T::BlockNumber, T::Hash>,
-			DealOrder<T::AccountId, T::BlockNumber, T::Hash, T::Moment>,
-		),
+		/// [funded_deal_order_id]
+		DealOrderFunded(DealOrderId<T::BlockNumber, T::Hash>),
 
 		/// A deal order has been locked by a borrower. This indicates that the borrower
 		/// is preparing to make a repayment and locks the loan from being sold or transferred
 		/// to another party.
-		DealOrderLocked(
-			DealOrderId<T::BlockNumber, T::Hash>,
-			DealOrder<T::AccountId, T::BlockNumber, T::Hash, T::Moment>,
-		),
+		/// [deal_order_id]
+		DealOrderLocked(DealOrderId<T::BlockNumber, T::Hash>),
```

Comment:
Assertion in `lock_deal_order_should_emit_deal_order_locked_event` doesn't match/assert on the arguments